test/test_nexus.py

```diff
@@ -1,16 +1,25 @@
 import time
 import os
+import uuid
 import pytest
 import logging
 import subprocess
 import signal
 
 from improv.nexus import Nexus
+from improv.store import Store
+
 
 # from improv.actor import Actor
 # from improv.store import Store
 
 SERVER_COUNTER = 0
+store_loc = str(os.path.join("/tmp/", str(uuid.uuid4())))
```

Comment:
You can actually import a test fixture from another file. That way, you could avoid repeating yourself here. I think I do this [here](https://github.com/project-improv/improv/pull/133/files#diff-b932c9512552ad7f3e29237447eebfe4fb81c86f7e9759d297b77107b266de9dR26) already.